src/app/store/index.js

```diff
@@ -43,7 +43,7 @@ export default new Vuex.Store({
   /**
    * If strict mode should be enabled
    */
-  strict: debug,
+  strict: false,
```

Comment:
Will keep this on `debug`. If the app is running in development environment, the strict mode will be set to true. This will improve code quality and makes you think more about the way Vuex works; you are not supposed to update the store outside of mutation handlers.
When setting `strict` to false, it is allowed to update the store any way you want, which is a bad practice.

More information can be found here:
https://vuex.vuejs.org/en/strict.html